graviti/dataset/dataset.py

```diff
@@ -8,7 +8,6 @@
 from typing import Any, Dict, Iterator, KeysView, Mapping, Optional
```

Comment:
file `openapi/request.py` is just committed, the code should be rebased and update `request.py` too.

Reply:
Okay